features/fields.feature

```diff
@@ -0,0 +1,341 @@
+#language: nl
+
+@post-assert
+Functionaliteit: Fields
+  Als consumer van de BRP API
+  Wil ik kunnen aangeven welke gegevens van de gevonden personen ik terug wil krijgen
+  Zodat ik alleen de gegevens terug krijg waarvoor ik doelbinding heb
+
+  Achtergrond:
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                             | waarde     |
+      | anummer                          | 2352984859 |
+      | burgerservicenummer              | 999995078  |
+      | geslachtsaanduiding (04.10)      | M          |
+      | geslachtsaanduiding.omschrijving | man        |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde   |
+      | geslachtsnaam         | Groenen  |
+      | voorvoegsel           | in den   |
+      | voornamen             | Franklin |
+      | aanduidingNaamgebruik | E        |

```

Comment:
Moet dit niet

> | aanduidingNaamgebruik.code | E |

zijn?

Reply:
ik heb overal de elementnummers toegevoegd. Dan weet de automation code dat hier aanduidingNaamgebruik.code bedoeld wordt, want dat is waar het de GBA waarde in komt